src/components/CountriesList.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react'
+
+
+import { Link } from 'react-router-dom/cjs/react-router-dom.min'
+import countries from "./countries.json"
+
+ 
+class CountriesList extends Component {
+    constructor() {
+        super()
+        this.state = { }
+    }
+
+
+    render() {
+ 
+        return (
+            <>
+                <div className="col-5" >
+                    <div className="list-group">
+                        {countries.map((elm, index) => <Link className="list-group-item list-group-item-action" to={elm.cca3}> {elm.flag}  {elm.name.common}</Link> )}

```

Comment:
```suggestion
                        {countries.map((elm, index) => <Link className="list-group-item list-group-item-action" to= to={`/${elm.cca3}`}> {elm.flag}  {elm.name.common}</Link> )}

```
Recuerda que la propiedad _to_ de [_Link_](https://reactrouter.com/web/api/Link) equivale al _href_ de un ancle, por lo que debes indicar la ruta a la que tiene que ir exacta. 

Como definiste en el Switch una ruta que esperaba después de / las iniciales del País, para que entre por la ruta, tienes que establecer el path del link con la misma estructura que lo haces en el path de la ruta